MobiusCore/Source/MobiusController.swift

```diff
@@ -303,6 +334,29 @@ public final class MobiusController<Model, Event, Effect> {
         }
     }
 
+    private func disconnect(uuid: UUID) {
+        do {
+            try state.mutate { stoppedState in
+                guard stoppedState.connectables[uuid] != nil else {
+                    throw ErrorMessage(message: "\(Self.debugTag): not connected, cannot disconnect")
+                }
+
+                stoppedState.connectables[uuid] = nil
+            }
+        } catch {
+            MobiusHooks.errorHandler(
+                errorMessage(error, default: "\(Self.debugTag): cannot disconnect while running; call stop first"),
+                #file,
+                #line
+            )
+        }
+    }
+
+    private static var viewConnectableID: UUID {
+        // Reserve 'nil' to prevent collision with generated connection ids
+        return UUID(uuid: (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
```

Comment:
Is this really necessary? Given that `UUID` is documented as (emphasis mine):

> A _universally unique_ value that can be used to identify types, interfaces, and other items.

I guess it doesn’t hurt, but?

Reply:
Yeah, it appears unnecessary but the reason behind it is that static stored properties aren't supported in types that use generics. This needs to return a consistent value, so I figured I might as well use [one](https://tools.ietf.org/html/rfc4122#section-4.1.7) that _shouldn't_ be generated otherwise.

Reply:
Thanks for the explanation. Makes sense to me, I'm good with keeping it personally.